### PR TITLE
Azure DevOps CI 

### PR DESCRIPTION
**Pull request by villagra**
Changes made:
- Generate appx bundle for all platforms: It doesn't affect the download size, the store client will download just the bits required for each platform and having all is no extra effort and will perform much better being native for the platform, specially ARM|ARM64. Do not remove them.
- Build just in Release config. It doesn't make sense to build in debug. Building in release will also generate test packages that can be sideloaded (folder Jellyfin_x.x.x_test)
- Added step to publish packages in the drop folder.
Notes:
- Do not rename the Jellyfin_TemporaryKey.pfx file. This is the name used in vs and the process to generate test certificates will always use that name.
- There is no autoincrement with each build, what I usually do it generate packages based in the number of commits. If you want to follow that approach, the mayor and minor will be automatically configured and the build will be the number of commits, something like: Jellyfin_1.1.3234

### CODE REVIEW

File: .ci/azure-pipelines.yml [hunk 1]
```diff
@@ -1,43 +1,36 @@
+# Universal Windows Platform
+# Build a Universal Windows Platform project using Visual Studio.
+# Add steps that test and distribute an app, save build artifacts, and more:
+# https://aka.ms/yaml
+
```
Comment (dkanada) [inline .ci/azure-pipelines.yml:5]: seems like this can be removed

File: .ci/azure-pipelines.yml [hunk 2]
```diff
@@ -1,43 +1,36 @@
...
-    - '*'
-  tags:
-    include:
-    - '*'
+- master
 
-pr:
-  branches:
-    include:
-    - '*'
```
Comment (dkanada) [inline .ci/azure-pipelines.yml:13]: We use these triggers for all projects so PRs and any branches will also get checked, can we revert this change?

File: .ci/azure-pipelines.yml [hunk 3]
```diff
@@ -1,43 +1,36 @@
...
-    displayName: 'Update DotNet'
-    inputs:
-      packageType: sdk
-      version: '3.1.100'
+- task: VSBuild@1
+  inputs:
+    platform: 'x86'
+    solution: '$(solution)'
+    configuration: '$(buildConfiguration)'
+    msbuildArgs: '/p:AppxBundlePlatforms="$(buildPlatform)" /p:AppxPackageDir="$(appxPackageDir)" /p:AppxBundle=Always /p:UapAppxPackageBuildMode=StoreUpload'
```
Comment (dkanada) [inline .ci/azure-pipelines.yml:30]: This is just for my knowledge, what's the difference between the `platform` input and the `AppxBundlePlatforms` variable passed to the build?

File: .ci/azure-pipelines.yml [hunk 4]
```diff
@@ -1,43 +1,36 @@
...
-      command: publish
-      publishWebProjects: false
-      projects: 'Jellyfin/Jellyfin.csproj'
-      arguments: '--configuration $(BuildConfiguration) --output $(Build.ArtifactStagingDirectory)'
-      zipAfterPublish: false
+- task: PublishBuildArtifacts@1
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+    ArtifactName: 'drop'
+    publishLocation: 'Container'
```
Comment (dkanada) [inline .ci/azure-pipelines.yml:36]:
```suggestion
    ArtifactName: 'jellyfin-uwp'
    publishLocation: 'Container'

```

Comment (dkanada) [reply .ci/azure-pipelines.yml:36]: Hmmm I wonder if that newline will count as an EOL newline or not.

File: .ci/azure-pipelines.yml (hunk 1 above)
Comment (villagra) [reply .ci/azure-pipelines.yml]: Removed

File: .ci/azure-pipelines.yml (hunk 2 above)
Comment (villagra) [reply .ci/azure-pipelines.yml]: Reverted

File: .ci/azure-pipelines.yml (hunk 3 above)
Comment (villagra) [reply .ci/azure-pipelines.yml]: platform is not used. It was a default value and I just removed it.
The project uses dotnet native (AOT), that means that it needs to be built for each supported platform, and AppxBundlePlatforms specify those.

File: .ci/azure-pipelines.yml (hunk 4 above)
Comment (villagra) [reply .ci/azure-pipelines.yml]: Updated artifactname

**Comment by EraYaN** [comment]
>     * There is no autoincrement with each build, what I usually do it generate packages based in the number of commits. If you want to follow that approach, the mayor and minor will be automatically configured and the build will be the number of commits, something like: Jellyfin_1.1.3234

The number of commits can change and is not strictly increasing, so we should probably try to stick to regular semver if at all possible. If you want to add something unique you can add the git hash to the InformationalVersion.

**Comment by dkanada** [comment]
I think incrementing manually with semver would be fine on its own.
